src/Resources/Locales/en.json

```diff
@@ -514,9 +514,8 @@
         "notImplemented": "Not implemented",
         "noPrimaryTwinProperties": "No primary twin properties to display",
         "twinAlias": {
-            "title": "Twin alias",
```

Comment:
please force a rev of the other languages too.

Reply:
I will I will :)